composer.json

```diff
@@ -15,14 +15,14 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": "^7.4|^8.0",
-        "symfony/console": "~3.0|~4.0|~5.0",
-        "symfony/dependency-injection": "~3.0|~4.0|~5.0",
-        "symfony/config": "~3.0|~4.0|~5.0",
-        "symfony/yaml": "~3.0|~4.0|~5.0",
-        "symfony/finder": "~3.0|~4.0|~5.0",
-        "symfony/filesystem": "~3.0|~4.0|~5.0",
-        "symfony/event-dispatcher": "~3.0|~4.0|~5.0",
+        "php": "^7.4|^8.0|^8.1",
+        "symfony/console": "~3.0|~4.0|~5.0|^6.1",
+        "symfony/dependency-injection": "~3.0|~4.0|~5.0|^6.1",
+        "symfony/config": "~3.0|~4.0|~5.0|^6.1",
+        "symfony/yaml": "~3.0|~4.0|~5.0|^6.1",
+        "symfony/finder": "~3.0|~4.0|~5.0|^6.1",
+        "symfony/filesystem": "~3.0|~4.0|~5.0|^6.1",
+        "symfony/event-dispatcher": "~3.0|~4.0|~5.0|^6.1",
```

Comment:
Can we also add tests against the new Symfony versions? They're defined in the [`php` workflow file](https://github.com/martin-helmich/typo3-typoscript-lint/blob/master/.github/workflows/php.yml).